src/docs/product/accounts/quotas/manage-event-stream-guide.mdx

```diff
@@ -1,175 +1,188 @@
 
-![Calculating rate limits](manage-event-stream-14.png)
+### Managing a Spike {#-spike-protection-was-activated----what-should-i-do}
 
-- Open the project DSN key configuration under `[Project Settings] > Client Keys > [Configure]`
-- Take a look at the `KEY USAGE IN THE LAST 30 DAYS` graph. Max daily rate in the last month is < 326K
-- Based on that, we can define a ceiling **daily** max value of ~330K, which is ~13,750 events an **hour**, or ~230 events a **minute**.
-- Notice that you can set a daily, hourly, or minute-based rate limit. We'd recommend using a minute based rate to avoid situations where a random event spike might exhaust your daily or hourly set quota and leave you blind for a long while.
-- You can always go back, check the graph to see the number of events dropped due to rate limiting and revisit your settings.
+You might receive an email notifying you that spike protection was triggered and applied to your account.
 
-![Revisit rate limits](manage-event-stream-15.png)
+![Spike Protection Email](spike-email.jpg)
 
-### > **How can I see a breakdown of incoming events?**
+Many times, an unexpected spike is caused by a new error (or errors) introduced into your code with a new release version. We understand that errors can come in spikes and service interruptions can be challenging, so we include a one-time grace period for all paid accounts.
 
-[Usage Stats](/product/stats/#usage-stats) displays details about the total number of events Sentry has received across your entire organization for up to 90 days. The report breaks down the events by project into three categories:
+If an account depletes its event capacity and has never previously triggered a grace period, we continue to accept events for the next three days at a maximum rate of 10,000 events per hour. If you choose not to increase your capacity, Sentry will stop accepting events for the remainder of the current billing month. However, if you increase your capacity, events accepted during the grace period will be counted as consumed capacity.
 
-- **Accepted**: Events processed and displayed in your event stream.
-- **Dropped**: Events that were discarded due to a limit being hit.
-- **Filtered**: Events that were blocked based on your inbound filter rules.
+Once you've used your grace period, we recommend using our on-demand spending caps to ensure you have time to adjust your capacity in the event of a future spike in errors.
 
-![Overview of Usage Stats page](../../stats/usage-stats.png)
+Also, consider doing the following:
 
-Clicking on the settings icon next to the project name will open the project's settings page where you can manage its Inbound Filters and Rate Limits.
+- Set better rate limits on the DSN keys for the projects related to the spike. - If it's a specific release version that has caused the spike, add the version identifier to the project's inbound filters to avoid accepting events from that release.
```

Comment:
```suggestion
- Set better rate limits on the DSN keys for the projects related to the spike. 
- If it's a specific release version that has caused the spike, add the version identifier to the project's inbound filters to avoid accepting events from that release.
```